presto-main/src/main/java/com/facebook/presto/execution/QueryExecution.java

```diff
@@ -45,7 +50,7 @@ Duration waitForStateChange(QueryState currentState, Duration maxWait)
 
     Session getSession();
 
-    void start(Optional<String> resourceGroupName);
```

Comment:
When we get request to fetch all queries for a particular user, QueryManager only has a QueryId -> QueryExecution map. So added a setter to QueryExecution to record ResourceGroupId. That way when we get a request we can go through all QueryExecution to find the ones that has matching user name, then get ResourceGroupId from QueryExecution.